packages/spotlight/src/electron/.cursor/rules/electron.mdc

```diff
@@ -0,0 +1,248 @@
+---
+description: Electron desktop app development guidelines
+globs:
+alwaysApply: true
+---
+
+# Electron App Guidelines
+
+The Spotlight desktop app packages the UI and server into a standalone Electron application.
+
+## Architecture
+
+```
+Electron App
+├── Main Process (Node.js)
+│   ├── electron/main/index.ts   # Window, tray, menus, auto-update
+│   ├── Runs sidecar server      # setupSpotlight() on port 8969
+│   └── IPC handlers             # Badge count, etc.
+│
+└── Renderer Process (Chromium)
+    ├── electron-index.tsx       # Entry point, Sentry init
+    └── UI (React app)           # Same as web UI, with IS_ELECTRON=true
+```
+
+## Main Process (`electron/main/index.ts`)
+
+The main process handles:
+
+### Window Management
+```typescript
+const win = new BrowserWindow({
+  titleBarStyle: 'hidden',           // Custom title bar
+  trafficLightPosition: { x: 16, y: 16 },  // macOS traffic lights
+  webPreferences: {
+    nodeIntegration: true,
+    sandbox: false,
+  },
+});
+
+// Load UI
+if (process.env.VITE_DEV_SERVER_URL) {
+  win.loadURL(process.env.VITE_DEV_SERVER_URL);  // Dev
+} else {
+  win.loadFile(path.join(__dirname, '../renderer/index.html'));  // Prod
+}
+```
+
+### Sidecar Server
+The main process runs the sidecar server internally:
+```typescript
+import { setupSpotlight } from '../../server/main';
+
+await setupSpotlight({
+  port: DEFAULT_PORT,  // 8969
+  incomingPayload: storeIncomingPayload,
+  isStandalone: true,
+});
+```
+
+### System Tray
+- Tray icon with context menu
+- Click to show/hide window
+- "Start at Login" option
+- Linux: No tray (quit on window close)
+
+### Auto-Update
+```typescript
+import { autoUpdater } from 'electron-updater';
+
+autoUpdater.checkForUpdates();
+autoUpdater.on('update-downloaded', () => {
+  // Prompt user to restart
+});
+```
+
+### Menus
+- Application menu (File, Edit, View, Window, Settings, Help)
+- Settings: "Send Error Reports", "Send Payload to Sentry", "Start at Login"
+
+### Persistent Settings
+```typescript
+import Store from 'electron-store';
+const store = new Store();
+
+store.get('sentry-enabled');
+store.set('sentry-enabled', true);
```

Comment:
Do we really have this or is this hallucination?

Reply:
we have it https://github.com/getsentry/spotlight/blob/7287f4b641c26405c6c3c6b361b0ab63998940db/packages/spotlight/src/electron/main/index.ts#L14 😆 

Persistent settings like "send error reports", "send payload to sentry"